monitoring_service.rst

```diff
@@ -2,145 +2,230 @@ Raiden Network Monitoring Service
 #################################
 
 
-Basic requirements for the MS
-=============================
-* Good enough uptime (a third party service to monitor the servers can be used to provide statistics)
-* Sybil Attack resistance (i.e. no one should be able to announce an unlimited number of (faulty) services)
-* Some degree of redundancy (ability to register balance proof with multiple competing monitoring services)
-* A stable and fast ethereum node connection (channel update transactions should be propagated ASAP as there is competition among the monitoring services)
-* If MS registry is used, a deposit will be required for registration
+Summary
+=======
+*   Monitoring Service (MS) listens to blinded Balance Proofs (BP) and proposed service fees in a public Matrix Room
+*   Any pre-registered MS can decide to monitor a channel and store corresponding BPs
+*   Then, whenever a channel is closed by calling `closeChannel` the MS will call `updateNonClosingBalanceProof` with
```

Comment:
It should be added that the client is expected to react first, so it gets a period to update itself.

Reply:
good point I will add "after x blocks" and we can include the number of blocks as soon we have decided